managing-os/templates/archlinux.md

```diff
@@ -16,27 +16,647 @@ be considered experimental as Qubes developers team use mainly Fedora-based VMs
+
+        *   **$ sudo qvm-run pass-io development 'cat /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/noarch/qubes-template.archlinux-3.0.4-201512290227.noarch.rpm' > /home/user/Templates/qubes-template-archlinux-3.0.4-201512290227.noarch.rpm**
+
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/cd-root-dom0_zps9zcdx6b9.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/cd-root-dom0_zps9zcdx6b9.png"/></a>
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/transfer-script-dom0_zpskyshvn4k.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/transfer-script-dom0_zpskyshvn4k.png"/></a>
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/transfer-tempalte_rpm-dom0_zpsxcfmfzif.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/transfer-tempalte_rpm-dom0_zpsxcfmfzif.png"/></a>
+<br>
+<br>
+<br>
+<br>
+
+<big>**If everything went correct there should be a Archlinux template listed in your Qubes VM Manager**</big>
+
+<br>
+<br>
+<br>
+---------------
+
+#**Package Manager Proxy Setup Section**
+
+
+One last thing to setup to have a "PROPERLY" functioning archlinux template.
+
+Archlinux package manager Pacman is  a fine package mangers execpt that we could not find a way to configure it to use the Qubes Update Proxy Service (QUPS) that would compliy with Qubes QUPS usage policy.  
+
+*If someone does find a way please post to the Qubes-Users or Devel google groups mailing list.*
+
+Powerpill is a full Pacman wrapper that not only give easy proxy configuration but futher offers numerous other advantages.   
+
+Please check out:
+
+[Archlinux Powerpill](https://wiki.archlinux.org/index.php/powerpill)
+
+[XYNE's (dev) Powerpill](http://xyne.archlinux.ca/projects/powerpill/)
+
+
+**Important Note:** Until Powerpill is configured you will have to open network access to the template to get the intial packages etc downloaded.  You can use the "allow full access for" a given time period in the FW settings of the template in the VMM or open up the various services thru the same window.  Remember to change it back if you choose the later route.  Actions needing network access will be noted with (needs network access)
+
+<br>
+<br>
+
+**1:  Editing Pacman's configuration file (pacman.conf)**
+
+*   Open archlinux terminal app
+
+*   edit /etc/pacman.conf
+
+*   **$ sudo nano -w /etc/pacman.conf**
+        
+* Below is the output of a correct pacman.conf file  Make the changes so your file matches this one or rename the orginal and create a new one and copy and paste this text into it.  Text should be justifed left in the file.  The changes from your default are to make gpg sig signing mandatory for packages but not required for DBs for the archlinux repos.  Also to add the repo (at the end) for the Powerpill package.
+
```

Comment:
I am having trouble with the formatting.  Because of the '#' use on each line I had to tab it in 4 tabs or use ``` to keep the format otherwise it would combine lines and increase font size.  But now with it in this way I can not bold the lines that have been changed per you suggestion or it reverts back for that line and ends up putting a few lines into one.  I am sure there is a way I am just too new to the git markdown format I do not know how to adjust so it works properly.  Suggestion?